src/components/SearchBar/SearchBar.styled.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+function sharedCss() {
+  return `
```

Comment:
use `css` function from styled component when returning just `css`. It evaluates your styles